Annotation/Arg.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Annotation;
+
+/**
+ * @Annotation
+ * @Target({"METHOD"})
+ * @Attributes({
+ *     @Attribute("value", type="string", required=true),
+ * })
+ */
+class Arg
+{
+    private $arg;
```

Comment:
This one is meant to be stand-alone and not a `ConfigurationInterface` implementation?